grouper/fe/settings.py

```diff
@@ -35,6 +37,14 @@ def __init__(self):
         self.shell = (
             [["/bin/false", "Shell support in Grouper has not been setup by the administrator"]],
         )
+        self.metadata_options = defaultdict(
```

Comment:
i really don't like defaultdict for production, but this is a personal bias.

I don't even think this is fixed by types because I _think_ it masquerades as a dict.